hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -389,291 +377,6 @@ func (mh *Memcached) implementingWebhooks() {
 	pkg.CheckError("adding imports", err)
 }
 
-// implementingController will customize the Controller
-func (mh *Memcached) implementingController() {
-	controllerPath := filepath.Join(mh.ctx.Dir, "controllers", fmt.Sprintf("%s_controller.go",
-		strings.ToLower(mh.ctx.Kind)))
-
-	// Add imports
-	err := kbutil.InsertCode(controllerPath,
-		"import (",
-		importsFragment)
-	pkg.CheckError("adding imports", err)
-
-	// Add RBAC permissions on top of reconcile
-	err = kbutil.InsertCode(controllerPath,
-		"/finalizers,verbs=update",
```

Comment:
Do we have this RBAC by default in the scaffold done by the plugin if not we need to add it on top